2-multitenant/modules/env_baseline/main.tf

```diff
@@ -19,6 +19,14 @@ locals {
   subnetworks_re     = "/subnetworks/([^/]*)$"
   projects_re        = "projects/([^/]*)/"
   cluster_project_id = data.google_project.eab_cluster_project.project_id
+  available_cidr_ranges = [
```

Comment:
@gtsorbo - Thoughts if the private node range should be defined in a different stage/location?  For example, test/setup and passed through?

Reply:
Commit: 
a5a6d4a externalizes the blocks to a variable, if we are to define this ranges in a different stage we can use the variable.

Reply:
Looks good - Thanks!